modules/platforms/dotnet/Apache.Ignite.Core.Tests/Client/Cluster/ClientClusterTests.cs

```diff
@@ -0,0 +1,228 @@
+﻿/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ * 
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite.Core.Tests.Client.Cluster
+{
+    using System;
+    using Apache.Ignite.Core.Cache.Configuration;
+    using Apache.Ignite.Core.Client;
+    using Apache.Ignite.Core.Configuration;
+    using Apache.Ignite.Core.Impl.Client.Cluster;
+    using NUnit.Framework;
+
+    /// <summary>
+    ///  Cluster API tests for thin client.
+    /// </summary>
+    public class ClientClusterTests : ClientTestBase
+    {
+        /** Cache name. */
+        private const string PersistentCache = "persistentCache";
+
+        /** Persistence data region name. */
+        private const string DataRegionName = "persistenceRegion";
+
+        /// <summary>
+        /// Sets up the test.
+        /// </summary>
+        [SetUp]
+        public override void TestSetUp()
+        {
+            var cacheCfg = new CacheConfiguration
+            {
+                Name = PersistentCache,
+                DataRegionName = DataRegionName
+            };
+
+            var ignite = Ignition.GetIgnite();
+            ignite.GetCluster().SetActive(true);
+            //to make sure there is no persisted cache from previous runs
```

Comment:
1. Blank lines before all comment lines
2. Space after `//`
3. Comments start with a capital letter

Reply:
Got it